contrib/cni/10-crio-bridge.conflist

```diff
@@ -16,5 +18,7 @@
             [{ "subnet": "10.85.0.0/16" }],
             [{ "subnet": "1100:200::/24" }]
```

Comment:
Add a gateway for  the route for the `1100:200::/24"`  range; omit the host, `::1` in the prefix too.  You probably want `::/64` and not `::/24 `as well.

```{ "dst": "1100:200::/64", "gw": "fe80::1" }```  or if there really is a router with addr `1100:200::1`
```{ "dst": "1100:200::/64", "gw": "1100:200::1" }```

Reply:
it is weird, the error comes from https://github.com/containernetworking/plugins/blob/ec76e3c35c55f3107c45a54ce310c9c7e98bac75/plugins/main/bridge/bridge.go#L961 , why it worked before?

Reply:
Perhaps the intention was for IPv6 default route ```::/0``` ?     Similar to how ```0.0.0.0/0``` is used for IPv4 

Then omitting gwy should be ok (this bridge `cni0` would be used as the gateway for the default IPv6 route.)

Reply:
yeah, I think that if you set `IsGateway` to true you have to omit the default routes

Reply:
Shall I add a route for the gateway or just set `IsGateway` to `false`?

Reply:
What was already there didn't/couldn't work.

Like v4, you need to use `default` destination.

```
                        "routes": [{
                                "dst": "0.0.0.0/0",
                                "dst": "::/0"
                        }],
```
works for me in cnitool, but 

```
        "routes": [
            { "dst": "0.0.0.0/0" },
            { "dst": "::/0" }
        ],
```

should as well

This plugin creates interfaces inside containers that point to this bridge `cni0` to communicate with other containers on this same bridge.  `routes` is used to add routes to the containers routing table, supplying `dst` to route to and `gwy `to reach that dst.  (`gwy` doesn't have to be this bridge, that's why it's required.) 

Using default for a route and `isGateway` is a shortcut that has the plugin create an IP addr for this bridge (  ` "bridge": "cni0"`) itself and use that as the gwy.

Reply:
That makes sense, thanks for sharing your insight it helps.